app/api/__init__.py

```diff
@@ -183,3 +185,10 @@
 api.route(DiscountCodeDetail, 'discount_code_detail', '/discount-codes/<int:id>')
 api.route(DiscountCodeRelationship, 'discount_code_event',
           '/discount-codes/<int:id>/relationships/event')
+
+# attendees
+api.route(AttendeeList, 'attendee_list', '/attendees', 'orders/<int:order_id>/attendees',
```

Comment:
Shouldn't this be `/orders/<int:order_id>/attendees` , a **/ ( forward slash ) ** in the beginning of the endpoint ?